andela/lotus-ah
# ##160451604 Users are able to like, dislike and unlike their articles

#### What does this PR do?
This is PR is to used to implement the like, dislike and unlike feature. It is also used to fetch all the liked and disliked articles.
   
#### Description of Task to be completed?
- like an article
- dislike an article
- unlike an article
- get all liked articles
- get all disliked articles

#### How should this be manually tested?
- Start the app locally on port `3000`
- Like an article via postman with the structure below
- Make a `POST` request to `/api/v1/articles/1/like`
- You should get a response with the structure below:
```js
{
    "status": "SUCCESS",
    "DBdata": {
        "id": 1,
        "userId": 1,
        "likes": true,
        "dislike": false,
        "articleId": 1,
        "updatedAt": "2018-10-16T20:35:48.713Z"
    },
    "message": "you liked the article"
}
```

#### Any background context you want to provide?
NA

#### What are the relevant pivotal tracker stories?
-[#160451604](https://www.pivotaltracker.com/story/show/160451604)
